### PR TITLE
Features/app detail sequence number

### PR DESCRIPTION
**Pull request by Divya-Kaavian**

### CODE REVIEW

File: db/schema.cds [hunk 1]
```diff
@@ -8,6 +10,7 @@ entity ApplicationDetail{
     SignatureSignedDate : Date not null;
     CreatedAt: Timestamp @cds.on.insert: $now;
     UpdatedAt: Timestamp @cds.on.insert: $now  @cds.on.update: $now;
+    ApplicationNumber : String;
```
Comment (Kokila-Kaavian) [inline db/schema.cds]: Add not null constraints

File: mta.yaml [hunk 2]
```diff
@@ -3,216 +3,219 @@ ID: dteConsentAppPortal
...
-- name: dteConsentAppPortal-srv
-  type: nodejs
-  path: gen/srv
-  requires:
-  - name: PostgreSQL-Reference
-  - name: dteConsentAppPortal-auth
-  - name: dteConsentAppPortal-destination-service
-  provides:
-  - name: srv-api
+  - name: dteConsentAppPortal-srv
```
Comment (Kokila-Kaavian) [inline mta.yaml]: Why mta file was changed. Check once

File: srv/handler/create-enrollment-form-action.js [hunk 3]
```diff
@@ -1,6 +1,7 @@
 const cds = require('@sap/cds');
 const { v4: uuidv4 } = require('uuid');
-const {emptyField} = require("./regex-and-error-message");
+const { emptyField } = require("./regex-and-error-message");
+const { string } = require('@sap/cds/lib/core/classes');
```
Comment (Kokila-Kaavian) [inline srv/handler/create-enrollment-form-action.js]: remove the un wanted require

File: srv/handler/create-enrollment-form-action.js [hunk 4]
```diff
@@ -12,6 +13,35 @@ const createEnrollmentFormDetail = async (req, entity, tx) => {
 
   try {
     const { ApplicationDetail, BuildingDetail, AccountDetail, ConsentDetail } = req?.data;
+    // Get the flag and prefix for the application number from environment variables
+    const prefixFlag = process.env.APPNUM_PREFIX_ENABLED;
+    const envTag = process.env.APPNUM_PREFIX;
```
Comment (Kokila-Kaavian) [inline srv/handler/create-enrollment-form-action.js]:
```suggestion
    const appNumPrefix = process.env.APPNUM_PREFIX;
```

**Review by Kokila-Kaavian — CHANGES_REQUESTED** [review]

File: db/schema.cds (hunk 1 above)
Comment (Divya-Kaavian) [reply db/schema.cds]: Done

File: mta.yaml (hunk 2 above)
Comment (Divya-Kaavian) [reply mta.yaml]: Updated

File: srv/handler/create-enrollment-form-action.js (hunk 3 above)
Comment (Divya-Kaavian) [reply srv/handler/create-enrollment-form-action.js]: Done

File: srv/handler/create-enrollment-form-action.js (hunk 4 above)
Comment (Divya-Kaavian) [reply srv/handler/create-enrollment-form-action.js]: Done

**Review by Kokila-Kaavian — APPROVED** [review]

**Pull request merged**